### PR TITLE
[PROCAPPS-858] Update Container Image Docs to include Trends Page

### PR DESCRIPTION
**Pull request by vatteh**
### What does this PR do? What is the motivation?
This pr will update the Container Images documentation to mention Container Image Trends and how to enable the feature.

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/infrastructure/containers/container_images.md [hunk 1]
```diff
@@ -22,6 +25,10 @@ The [container images view][1] in Datadog provides key insights into every image
 
 {{< img src="security/vulnerabilities/container_images.png" alt="The container images view highlighting vulnerabilities and container column sort feature" width="100%">}}
 
+The [container image trends view][9] provides high-level insights across all of your images in your containerized infrastructure. Container Image Trends metrics can help you answer key questions about your image posture and deployment footprint over the span of weeks and months.
```
[the remark this answers is not in the document]
Comment (badovm) [reply content/en/infrastructure/containers/container_images.md:28]: +1

File: content/en/infrastructure/containers/container_images.md [hunk 2]
```diff
@@ -151,6 +158,12 @@ container_image:
 
 Set up the [AWS integration][4] to begin crawling Container Image metadata from Amazon ECR.
 
+## Configure container images trends view
+
+Image metrics on the Container Image Trends view are collected from the Live Containers and Image Collection source. The same instructions above allow the collection of container image metrics.
```
Comment (badovm) [inline content/en/infrastructure/containers/container_images.md:163]:
```suggestion
Live Containers and Image Check sources
```

Comment (badovm) [inline content/en/infrastructure/containers/container_images.md:163]:
```suggestion
Image metrics on the Container Image Trends view are collected from the Live Containers and Image Check sources. The same instructions above allow the collection of container image metrics.
```

File: content/en/infrastructure/containers/container_images.md [hunk 3]
```diff
@@ -151,6 +158,12 @@ container_image:
 
 Set up the [AWS integration][4] to begin crawling Container Image metadata from Amazon ECR.
 
+## Configure container images trends view
```
[the remark this answers is not in the document]
Comment (vatteh) [reply content/en/infrastructure/containers/container_images.md:161]: Tried to make the title similar to the `## Configure container images view` in ln 29, but not too opinionated on this

File: content/en/infrastructure/containers/container_images.md [hunk 4]
```diff
@@ -22,6 +22,10 @@ The [container images view][1] in Datadog provides key insights into every image
 
 {{< img src="security/vulnerabilities/container_images.png" alt="The container images view highlighting vulnerabilities and container column sort feature" width="100%">}}
 
+The [container image trends view][9] provides high-level insights across all of your images in your containerized infrastructure. Container Image Trends metrics can help you answer key questions about your security posture and deployment footprint over the span of weeks and months.
```
Comment (urseberry) [inline content/en/infrastructure/containers/container_images.md:25]:
```suggestion
The [container image trends view][9] provides high-level insights across all of your images in your containerized infrastructure. Container image trends metrics can help you answer key questions about your security posture and deployment footprint over the span of weeks and months.
```

File: content/en/infrastructure/containers/container_images.md [hunk 5]
```diff
@@ -151,6 +155,14 @@ container_image:
 
 Set up the [AWS integration][4] to begin crawling Container Image metadata from Amazon ECR.
 
+## Configure container images trends
+
+Use the Container Image Trends Configuration modal and toggle 'Enable Container Image Metric Collection' to turn on the generation of image metrics.
```
Comment (urseberry) [inline content/en/infrastructure/containers/container_images.md:160]:
```suggestion
Use the container image trends configuration modal and toggle **Enable Container Image Metric Collection** to turn on image metric generation.
```

File: content/en/infrastructure/containers/container_images.md [hunk 6]
```diff
@@ -151,6 +155,14 @@ container_image:
...
+
+Image metrics are collected from the [Live Containers](#live-containers) and [Image Check](#image-collection) sources. Follow the same instructions above to ensure that these collections are enabled across your entire infrastructure and take full advantage of the Trends view.
```
Comment (urseberry) [inline content/en/infrastructure/containers/container_images.md:162]:
```suggestion
Image metrics are collected from the [Live Containers](#live-containers) and [Image Check](#image-collection) sources. Follow the same instructions as above to ensure that these collections are enabled across your entire infrastructure and take full advantage of the trends view.
```

**Pull request merged**